docs/content/latest/api/ysql/syntax_resources/ysql_grammar.ebnf

```diff
@@ -538,3 +612,21 @@ param_name ::= '<Text Literal>' ;
 param_value ::= '<Text Literal>' ;
 
 code ::= '<Text Literal>' ;
+
+aggregate_argmode = 'IN' | 'VARIADIC' ;
```

Comment:
::= 
Same for others below.

Reply:
See issue #2308 for the move towards `=`.  Feel free to add comments there if you still prefer `::=`.